### PR TITLE
docs: change in how to write professional commits

### PR DESCRIPTION
**Pull request by suzy-g38**
## Fixes Issue

I have added examples of some commit messages as proposed in issue number #139.

In addition, 
1. I explained the conventional way of writing commit messages (with the structure).
2.  Added the proper git command( how the conventional way will look in the git command format),
3. Added "Rules & Specification" heading to points that were there previously at the beginning of the page and added
    three extra points.
4. Rename the "here are some examples of good commit messages" to "Type of Commits".

## Screenshots

image
image
image
image

## Note to reviewers
I hope you will find these additional changes helpful. But if not, I will make the changes accordingly. 
Thank you.

### CODE REVIEW

File: docs/guides/general-terminology/How-to-write-professional-commits.mdx [hunk 1]
```diff
@@ -4,23 +4,66 @@ title: How to write professional commits
...
+- `type` : Commit Type like build, ci, docs, feat, fix etc. This field is mandatory.
+
+- `scope` : Phrase describing a section of the codebase. This field is optional.
+
+- `description` : description should be in the present tense. Not capitalized. No period in the end. Each description consists of a Title, Body and Footer. The Title and Body are mandatory if the description is large enough. The footer is optional.
+
+#### The command :
+
+git commit -m "`type_of_commit(optional scope)`: `description_title/description`" -m "`description body(optional)`" -m "`description footer(optional)`"
+
```
Comment (priyankarpal) [inline docs/guides/general-terminology/How-to-write-professional-commits.mdx:22]: We can add an example here.

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
Looks Good 🎉

File: docs/guides/general-terminology/How-to-write-professional-commits.mdx [hunk 2]
```diff
@@ -4,23 +4,70 @@ title: How to write professional commits
...
+#### The Example:
+
+git commit -m "`docs(guides)`: `change in how to write professional commit`" -m "`fix spelling mistake, add example`" -m "`fixes issue #120`"
```
Comment (priyankarpal) [inline docs/guides/general-terminology/How-to-write-professional-commits.mdx:25]: This is what we wanted as an example.
Looks Good 🎉

**Review by suzy-g38 — COMMENTED** [review]
I have added the example, please review that.

File: docs/guides/general-terminology/How-to-write-professional-commits.mdx (hunk 2 above)
Comment (suzy-g38) [reply docs/guides/general-terminology/How-to-write-professional-commits.mdx:25]: > This is what we wanted as an example. Looks Good 🎉

I am really glad that you liked it.

**Comment by priyankarpal** [comment]
> I have added the example, please review that.



> I have added the example, please review that.

I already checked. It's great.

**Comment by suzy-g38** [comment]
> > I have added the example, please review that.
> 
> I have already checked. It's great.

Thank you. I am really glad.

**Pull request merged**